docs/app/documentation/component-docs/combobox-input/combobox-input-header/combobox-input-header.component.html

```diff
@@ -1,7 +1,8 @@
 <header>Combobox Input</header>
 <description>
     The combobox input component is an opinionated composition of the "input-group" and "popover" components to
-    accomplish the UI pattern for a typical input dropdown, also known as combobox input.
+    accomplish the UI pattern for a typical input dropdown, also known as combobox input. There is a way to hide last
+    + New item, by not observe the <code>(newItemClicked)</code> event.
```

Comment:
To hide the <code>+ New Item</code>, simply stop listening to the <code>(newItemClicked)</code> event.